### PR TITLE
add counter

### PR DESCRIPTION
**Pull request by augustinanders**
UserStory 16 #27

### CODE REVIEW

File: pages/twoplayer/index.js [hunk 1]
```diff
@@ -1,14 +1,28 @@
...
 import TwoPlayerGrid from "../../components/TwoPlayerGrid";
+import StyledDoubleSection from "../../components/StyledDoubleSection";
 import store from "../../zustand/store";
 
 export default function TwoPlayerPage() {
   const player = store((state) => state.player);
+  const matchesPlayerOne = store((state) => state.matchesPlayerOne);
+  const matchesPlayerTwo = store((state) => state.matchesPlayerTwo);
+
+  console.log(matchesPlayerOne, matchesPlayerTwo);
```
Comment (onemanwenttomow) [inline pages/twoplayer/index.js:12]: I think we would want to remove console logs before merging into main.

Comment (augustinanders) [reply pages/twoplayer/index.js]: 👍 thanks

File: components/TwoPlayerGrid/index.js [hunk 2]
```diff
@@ -5,9 +5,13 @@ import MemoryGrid from "../MemoryGrid";
 export default function SinglePlayerGrid() {
   const setPlayer = store((state) => state.setPlayer);
   const player = store((state) => state.player);
+  const setMatches = store((state) => state.setMatches);
 
   const { handleReveal, handleConceal, shuffledImages, compareImages } =
     useMemoryGame({
+      onMatchSolved: () => {
```
Comment (onemanwenttomow) [inline components/TwoPlayerGrid/index.js:12]: I think we could shorten this a little to the following. 

``` js
onMatchSolved: () => {
  setMatches();
},
```

to

`onMatchSolved: setMatches,`

**Comment by onemanwenttomow** [comment]
Couple of minor comments otherwise LGTM

**Pull request merged**